indico/modules/events/client/js/display.js

```diff
@@ -5,9 +5,13 @@
 // modify it under the terms of the MIT License; see the
 // LICENSE file for more details.
 
+/* global ajaxDialog */
+
 import React from 'react';
 import ReactDOM from 'react-dom';
 
+import 'indico/custom_elements/ind_collapsible';
+import 'indico/custom_elements/ind_with_tooltip';
```

Comment:
I usually keep imports with side effects at the end of the imports (eslint cannot sort them because it does not know if they have to be at a specific position or not)